cmd/zetacored/parse_genesis.go

```diff
@@ -124,7 +141,7 @@ func ImportDataIntoFile(genDoc *types.GenesisDoc, importFile *types.GenesisDoc,
 		if Copy[m] {
 			appState[m] = importAppState[m]
 		}
-		if Modify[m] {
+		if Modify[m] && modifyEnabled {
```

Comment:
Disabling modify as some of the proto files have changed since v17. 
Examples 
`inbound_tx_params`  -> `inbound_params`
`OutBoundTx` -> `OutboundTx`